ansible_ai_connect/users/views.py

```diff
@@ -228,5 +229,7 @@ def post(self, request, *args, **kwargs):
             event.set_request(request)
             send_schema1_event(event)
 
+            schema2.oneclick_trial_event_send(request.user)
```

Comment:
As mentioned above, I would prefer if this was done in two different operations, one to get the correct payload, and a second one for the transport. This to avoid a dependency hell where our payload validation part ends up loading the Segment client that it doesn't need.

Reply:
oh I see what you mean now! about my comment above!

Reply:
yes I was thinkkng on just adding here 2 operations, get the payload and send the event, but I was at the end tyring to encapsulate mehtods for reusing. Although it is true that it provides less flexibility, it is not maybe necessary

Reply:
Ignore my comment, I didn't realize we had two different files (schema2.py and schema2_utils.py).

Reply:
thanks , anyway it is great brainstorming about this! It is always good to care about code <3